### DISCUSSION TITLE
migrate DBA Dash from one server to another

### DISCUSSION THREAD

**Discussion by IndrajitMurgude** [trimble-oss/dba-dash#809] [Q&A] [answered]
Hi, I wanted to migrate DBA Dash tool along with repository database from one server to another new server.
Right now, I have 10 server already being monitored, when I migrate the tool to new server, I am going to add few more new server to monitoring along with existing 10 server. I also do not want to loose the data being collected in repository database.
Are there any defined steps for this kind of migration ?
Thank you !

**Comment by DavidWiseman** [comment, marked as answer]
## To move the repository DB:

* Use backup/restore to move the database to the new server.
* Ensure the service account has access to the DB on the new server.
* Change the destination connection in the service config tool to point to the new instance.

## To move the service:

* Copy/Paste the app folder from one server to the other.  Or download a fresh copy and transfer the **ServiceConfig.json** file.  All configuration made using the config tool is in this file.  You might also have a **commandTimeouts.json** file and a **PerformanceCountersCustom.xml** file if you customized timeouts or performance counters.

* If the config is encrypted, you will need to supply the password when you load the config tool on the new server.  

* You will need to install as a service.  If you use the same account as the old server that will make things easier with permissions.  Or you will need to ensure the new service account has permissions to your monitored instances.  Using a group managed service account is a good option.

* Ensure the service account has permissions to write to the app folder on the new server.  Limit access to other accounts to this folder.

* You can then stop the service on the old server and start it on the new server.  If everything works as expected, click the link to uninstall the service on the old server.

Hope this helps.
